lib/src/consts.rs

```diff
@@ -135,6 +135,15 @@ impl Default for Eip1559Constants {
     }
 }
 
+#[repr(u8)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
+pub enum VerifierType {
```

Comment:
Could just reuse ProofType?

Reply:
ProofType is in host, this thing we gotta use in lib. Reusing will cause circular dep

Reply:
Move ProofType to lib?

Reply:
Don't think it's possible cuz its impl contains lots of host content